apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -705,14 +701,22 @@ contract_transactions(_Config) ->
                                fun get_contract_call/1,
                                fun aect_call_tx:new/1, MinerPubkey),
 
+    {ok, 200, #{<<"tx">> := EncodedUnsignedContractCallTx,
+                <<"tx_hash">> := ContractCallTxHash}} = get_contract_call(ContractCallEncoded),
+    sign_and_post_tx(EncodedUnsignedContractCallTx),
+
+    % mine a block
+    aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE), 2),
+    tx_is_mined_test(MinerAddress, ContractCallTxHash),
```

Comment:
I see contract call tx with already encoded call data was not mined in chain - now it is posted to node and mined in chain.